packages/web-pkg/src/composables/useAppDefaults/useClient.ts

```diff
@@ -0,0 +1,8 @@
+import { getCurrentInstance } from '@vue/composition-api'
+
+// FIXME: use proper type
+export type Client = any
+
+export const useClient = (): Client => {
+  return (getCurrentInstance().proxy as unknown as any).$client
```

Comment:
The files app has a `ClientService`, I'd like to utilize that here. Would need to be moved to `web-pkg` first, obviously. What do you think?